week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,19 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
-}
+};
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+    return "Hello, my name is " + name + " and I am " +
+        age + " years old";
+};
 
 function getRemainder(a, b) {
-    remainder = a %% b;
-
+    let remainder = a % b;
```

Comment:
good catch Orhan, declaring the variable reminder is something important, as without it you are making it a global variable. The only thing is you can use const here, as you're not planning to reassign reminder after that